src/org/traccar/WebDataHandler.java

```diff
@@ -121,10 +129,50 @@ public String formatRequest(Position position) {
 
     @Override
     protected Position handlePosition(Position position) {
+        if (json) {
+            AsyncHttpClient.BoundRequestBuilder requestBuilder = Context.getAsyncHttpClient().preparePost(url);
+            requestBuilder.setBodyEncoding(StandardCharsets.UTF_8.name());
+
+            requestBuilder.addHeader("Content-Type", getContentType());
 
-        Context.getAsyncHttpClient().prepareGet(formatRequest(position)).execute();
+            setContent(position, requestBuilder);
+            requestBuilder.execute();
 
+        } else {
+            Context.getAsyncHttpClient().prepareGet(formatRequest(position)).execute();
+        }
         return position;
     }
 
+    protected String getContentType() {
+        return "application/json; charset=utf-8";
+    }
+
+    protected void setContent(Position position, AsyncHttpClient.BoundRequestBuilder requestBuilder) {
```

Comment:
Same question.

Reply:
Here, we need a separate method because I don't have Event object in WebDataHandler, so that method is edited accordingly

Reply:
I don't understand what events have to do with this functionality.

Reply:
What did you mean by separate method ? maybe I dont get it.